site/public/js/manage-grades.js

```diff
@@ -0,0 +1,68 @@
+/* exported toggleColumnsForm, updateManageStudentsColumns, fillAllCheckboxes
```

Comment:
add the function `updateManageGradesColumns` to the exported message to fix the linting error